modules/ROOT/pages/configure-api-task.adoc

```diff
@@ -25,19 +25,25 @@ image::apiconfig.png[height=321,width=321]
 +
 . Above API Configuration, in Add Consumer Endpoint, optionally enter the public-facing URL for the API. For example, enter `+http://product.mydomain.com+`.
 +
-. In Managing Type, choose Basic Endpoint or Endpoint with Proxy.
+. Browse to the bottom of the page and in *Managing Type*, choose Basic Endpoint or Endpoint with Proxy.
 +
-* In Implementation URI, optionally enter the URL of the site that hosts the API or accept the default, a dummy URI.
+. Check the *Mule version* box if you want to manage API in Mule 4 or above.
```

Comment:
```suggestion
. Select the *Mule version* box if you want to manage APIs in Mule 4 or later.
```
GLOBALLY change "below" and "above" in versions to "earlier" and "later," per Edit Tips and the style ref.

Reply:
Will do! Thanks!